plugins/jira/commands/solve.md

```
@@ -70,7 +70,18 @@ This command takes a JIRA URL, fetches the issue description and requirements, a
     - Ensure godoc comments are generated for any newly created public functions
       - Use your best judgement if godoc comments are needed for private functions
       - For example, a comment should not be generated for a simple function like func add(int a, b) int { return a + b}
-    - Create unit tests for any newly created functions
+    - Create unit tests for any newly created functions (see Step 3.5 for details)
```

markdownlint-cli2 (0.23.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 3; Actual: 4

(MD005, list-indent)